diff --git a/btusb.c b/btusb.c
index 50df417..875ef8a 100644
--- a/btusb.c
+++ b/btusb.c
@@ -27,7 +27,7 @@
 #include "btrtl.h"
 #include "btmtk.h"
 
-#define VERSION "0.8"
+#define VERSION "1.8"
 
 static bool disable_scofix;
 static bool force_scofix;
@@ -254,6 +254,10 @@ static const struct usb_device_id blacklist_table[] = {
 	{ USB_DEVICE(0x0489, 0xe036), .driver_info = BTUSB_ATH3012 },
 	{ USB_DEVICE(0x0489, 0xe03c), .driver_info = BTUSB_ATH3012 },
 
+	/* Nintendo Wii Bluetooth module */
+	/* Blacklist to prevent interfering with Dolphin */
+	{ USB_DEVICE(0x057e, 0x0305), .driver_info = BTUSB_IGNORE },
+
 	/* QCA ROME chipset */
 	{ USB_DEVICE(0x0cf3, 0x535b), .driver_info = BTUSB_QCA_ROME |
 						     BTUSB_WIDEBAND_SPEECH },
